### ISSUE TITLE
Wrong initial state while setting initialPage in PageController

### ISSUE THREAD

**Issue by hiking93** [CaiJingLong/flutter_page_indicator#1]
When `initialPage` is set with value other than 0, page indicator still indicates page 0 before gesture events.
```
PageIndicatorContainer(
  pageView: PageView.builder(
    controller: PageController(initialPage: index),
    itemCount: media.length,
    itemBuilder: (context, index) => Stack(
          fit: StackFit.expand,
          children: [
            Center(child: CircularProgressIndicator()),
            FadeInImage.memoryNetwork(
                placeholder: kTransparentImage,
                image: media[index].url,
                fadeInDuration: Duration(milliseconds: 300),
                fit: BoxFit.contain),
          ],
        ),
    onPageChanged: _onPageChanged,
  ),
  length: media.length,
  padding: EdgeInsets.only(bottom: 8),
  indicatorColor: Color(0xaaaaaaaa),
  indicatorSelectorColor: Theme.of(context).accentColor,
  size: 8,
),
```

**Comment by CaiJingLong** [comment]
I'll check it out. If there's a problem, I'll fix it as soon as possible.

**Comment by CaiJingLong** [comment]
fixed in #2 
use version `0.1.3`
